applications/dashboard/views/log/confirm.php

```diff
@@ -57,14 +57,31 @@
                     <?php
                     if (count($this->data('Users')) > 1) {
                         echo '<div class="CheckBoxCell">';
-                        echo $this->Form->CheckBox('SelectAll', t('All'));
+                        echo wrap(
+                            $this->Form->checkBox(
+                                'SelectAll',
+                                t('All'),
+                                ['display' => 'after', 'class' => 'js-check-all checkbox checkbox-inline']
+                            ).' '.t('All'),
+                            'span',
+                            ['class' => 'checkbox-painted-wrapper']
+                        );
+
                         echo '</div>';
                     }
 
                     foreach ($this->data('Users') as $User) {
                         $RecordUser = Gdn::userModel()->getID($User['UserID'], DATASET_TYPE_ARRAY);
                         echo '<div class="CheckBoxCell">';
-                        echo $this->Form->CheckBox('UserID[]', htmlspecialchars($User['Name']), array('value' => $User['UserID'], 'class' => 'checkbox checkbox-inline'));
+                        echo wrap(
+                            $this->Form->checkBox(
+                                'UserID[]',
+                                htmlspecialchars($User['Name']),
+                                array('value' => $User['UserID'], 'display' => 'after', 'class' => 'js-check-me checkbox checkbox-inline')
```

Comment:
While you're already in here, we can go ahead and update this array to use the short syntax.